man/docker-run.1.md

```diff
@@ -478,7 +478,8 @@ any options, the systems uses the following options:
 
 **-v**, **--volume**=[] Create a bind mount
```

Comment:
@moxiegirl man/docker-create.1.md has a very basic description of volume. It does not even specify options ro/rw or z/Z. So it does not look like it has kept pace with docker-run.1.md.That's why I left it untouched.

I will change the --volume description as you have suggested.